### ISSUE TITLE
The redundantLet rule can break SwiftUI views that use let _ = print()

### ISSUE THREAD

**Issue by danniss10** [nicklockwood/SwiftFormat#1127]
Say I have a swiftUI view that looks like the following: 
```
HStack {
    let _ = print("Hi")
    Text("Some text")
}
```

The redundantLet rule will change the code to this:
```
HStack {
    _ = print("Hi")
    Text("Some text")
}
```

That code will fail to build with a compile time error: 

> Type '()' cannot conform to 'View'

**Comment by nicklockwood** [comment]
@danniss10 fixed in 0.49.3

**Comment by danniss10** [comment]
Thank you @nicklockwood!
